[core]/es_extended/server/main.lua

```diff
@@ -628,6 +628,22 @@ ESX.RegisterServerCallback('esx:getPlayerNames', function(source, cb, players)
   cb(players)
 end)
 
+ESX.RegisterServerCallback("esx:spawnVehicle", function(source,cb,vehData)
+    local ped = GetPlayerPed(source)
+    ESX.OneSync.SpawnVehicle(vehData.model or `ADDER`, vehData.coords or GetEntityCoords(ped), vehData.coords.w or 0.0, vehData.props or {}, function(id)
+        local vehicle = NetworkGetEntityFromNetworkId(id)
```

Comment:
If vehData.wrap true  only use 

local vehicle = NetworkGetEntityFromNetworkId(id)

please put this if statement  thank you.

Reply:
@Taavi-AU Please do this change.